projects/Mallard/src/components/article/html/components/header.ts

```diff
@@ -355,13 +365,13 @@ export const headerStyles = ({ colors, wrapLayout, theme }: CssProps) => css`
         font-family: ${families.titlepiece.regular};
     }
 
-    /*picture*/
+    /*gallery*/
     .header-container[data-type='${
-        ArticleType.Picture
```

Comment:
Why is Picture itself going away here?

Reply:
if you look at #738 I added it (we didn't have it) just to render this header - this was hacky, now that we have galleries, which are a real article type, I can repurpose this header as the gallery header